packages/browser-wallet/src/background/web3Id.ts

```diff
@@ -126,12 +128,21 @@ function rejectRequest(message: string): { run: false; response: MessageStatusWr
     };
 }
 
+// TODO Expose function from SDK and re-use.
+function timestampToDate(attribute: TimestampAttribute): Date {
+    return new Date(Date.parse(attribute.timestamp));
+}
+
 function validateTimestampAttribute(attributeTag: string, attributeValue: AttributeType) {
-    if (
-        attributeValue instanceof Date &&
-        (attributeValue.getTime() < MIN_DATE_TIMESTAMP || attributeValue.getTime() > MAX_DATE_TIMESTAMP)
-    ) {
-        return `The attribute [${attributeValue}] for key [${attributeTag}] is out of bounds for a Date. The Date must be between ${MIN_DATE_ISO} and ${MAX_DATE_ISO}`;
+    if (isTimestampAttribute(attributeValue)) {
+        const timestamp = timestampToDate(attributeValue).getTime();
+        if (Number.isNaN(timestamp)) {
+            return `The attribute [${attributeValue.timestamp}] for key [${attributeTag}] cannot be parsed as a Date.`;
+        }
```

Comment:
Looks good, only issue i can find is that this allows for Non-standard parsing. (like date "1000")
But we can probably live with that.

Reply:
I agree we can live with that.